appeals/web/src/server/appeals/appeal-details/__tests__/appeal-details.test.js

```diff
@@ -1949,4 +1949,40 @@ describe('appeal-details', () => {
 
 		expect(element.innerHTML).toMatchSnapshot();
 	});
+
+	it('should render "change" action links for appeal with LPAQ status "received"', async () => {
+		const appealId = '3';
+
+		nock('http://test/')
+			.get(`/appeals/${appealId}`)
+			.reply(200, {
+				...appealData,
+				appealId,
+				documentationSummary: {
+					appellantCase: {
+						status: 'received',
+						dueDate: '2024-10-02T10:27:06.626Z',
+						receivedAt: '2024-08-02T10:27:06.626Z'
+					},
+					lpaQuestionnaire: {
+						status: 'received',
+						dueDate: '2024-10-11T10:27:06.626Z',
+						receivedAt: '2024-08-02T10:27:06.626Z'
+					}
+				}
+			});
+
+		const response = await request.get(`${baseUrl}/${appealId}`);
+
+		const unprettifiedElement = parseHtml(response.text, { skipPrettyPrint: true });
+		expect(unprettifiedElement.innerHTML).toContain(
+			`href="/appeals-service/appeal-details/3/inspector-access/change/lpa"`
+		);
+		expect(unprettifiedElement.innerHTML).toContain(
+			`href="/appeals-service/appeal-details/3/neighbouring-sites/change/affected"`
+		);
+		expect(unprettifiedElement.innerHTML).toContain(
+			`href="/appeals-service/appeal-details/3/safety-risks/change/lpa"`
+		);
+	});
```

Comment:
Ideally there should also be a unit test to check that the change links aren't rendered when the LPAQ hasn't been recieved

Reply:
done